README.md

```diff
@@ -29,10 +32,18 @@ And the following are optional:
 * `HUBOT_IRC_PRIVATE`
 * `HUBOT_IRC_USESASL`
 
+| Environment Variable | Description |
+| -------------------- | ----------- |
+| `BUGZILLA_BLOCKER_PRODUCT` | A Bugzilla product (e.g., Taskcluster). |
+| `BUGZILLA_BLOCKER_TIME_PERIOD` | The waiting interval (in milliseconds) before examining blocker bugs. If omitted it will default to 5 minutes. |
+
 For more details on accepted environment variables, refer
 to [hubot-irc#configuring-the-adapter](https://github.com/nandub/hubot-irc/blob/master/README.md#configuring-the-adapter).
 
-_Example: Configuring for mozilla's taskcluster-bots IRC channel_
+To get started local development, create a `.env` file in the root of the repo.
```

Comment:
You don't *have* to create a `.env` file, these can be set in the environment as well.